app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Elf(Player, ABC):

```

Comment:
And here.

Reply:
well I just put ABC there because of that https://prnt.sc/rqiCg0VS0CZL
and in the task Elf and Dwarf were also mentioned to be abstract, so..